src/Features/Core/Portable/CodeFixes/Suppression/ISuppressionFixProvider.cs

```diff
@@ -11,14 +11,21 @@ namespace Microsoft.CodeAnalysis.CodeFixes.Suppression
     internal interface ISuppressionFixProvider
     {
         /// <summary>
-        /// Returns true if the given diagnostic can be suppressed.
+        /// Returns true if the given diagnostic can be suppressed or triaged.
         /// </summary>
         bool CanBeSuppressed(Diagnostic diagnostic);
 
         /// <summary>
-        /// Gets one or more suppression fixes for the specified diagnostics represented as a list of <see cref="CodeAction"/>'s.
+        /// Gets one or more suppression or triage fixes for the specified diagnostics represented as a list of <see cref="CodeAction"/>'s.
```

Comment:
I guess this is ok since the functions are internal - but this could cause confusion if triaging is not implemented soon.